plugins/GeoIp2/LocationProvider/GeoIp2.php

```diff
@@ -136,12 +146,12 @@ public static function getPathForGeoIpDatabase($filename)
      *
      * @return array eg. array('1.2.3.4', array(self::COUNTRY_CODE_KEY => ...))
      */
-    private static function getTestIpAndResult()
+    protected function getTestIpAndResult()
     {
         static $result = null;
         if (is_null($result)) {
             $expected = array(self::COUNTRY_CODE_KEY => 'FR',
-                self::REGION_CODE_KEY  => 'BFC',
+                self::REGION_CODE_KEY => 'BFC',
```

Comment:
The DB-IP Lite database returns no region for this IP, so the provider is reported as broken.
In relation to #15317 we maybe shouldn't use only one IP to check if the results are ok. Maybe we could have one IP per continent and go through the list until one matches exactly? Any idea where to get some IPs with static results?

Reply:
Or we might use different ips/results depending on the provider

Reply:
What about using the test database we include for system tests? Could we load that so we can control the IP/location? I started on supporting multiple ip addresses, but we'd have to have quite a few and they might change...

Reply:
I guess that wouldn't work for server modules... though it looks like servermodule overrides isWorking(). If we move that logic to the Php class it might work I think.

Reply:
I think I'm going to make the check less strict and just check that there's at least some data in the provider. Users can always see their own IP geolocated to check for accuracy.